src/bsoncxx/test/v_noabi/decimal128.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2009-present MongoDB, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <bsoncxx/decimal128.hpp>
+
+//
+
+#include <bsoncxx/test/v1/decimal128.hh>
+
+#include <bsoncxx/exception/error_code.hpp>
+#include <bsoncxx/exception/exception.hpp>
+#include <bsoncxx/stdx/string_view.hpp>
+
+#include <bsoncxx/test/catch.hh>
+
+namespace {
+
+using bsoncxx::v_noabi::decimal128;
+using code = bsoncxx::v_noabi::error_code;
+
+TEST_CASE("exceptions", "[bsoncxx][v_noabi][decimal128]") {
```

Comment:
Some minimal test coverage was added to v\_noabi to ensure backward compatibility.